package.json

```diff
@@ -14,6 +14,7 @@
     "babel-preset-env": "^1.4.0",
     "babel-preset-react": "^6.24.1",
     "babel-preset-react-native": "^1.9.1",
+    "babel-plugin-transform-strict-mode": "^6.22.0",
```

Comment:
Can we use version `6.24.1`? We already have that in our transitive deps (see `yarn.lock`)